experiments/prototype_omip_simulation/Manifest.toml

```diff
@@ -1304,9 +1304,9 @@ version = "0.5.5"
 
 [[deps.Oceananigans]]
 deps = ["Adapt", "CUDA", "Crayons", "CubedSphere", "Dates", "Distances", "DocStringExtensions", "FFTW", "Glob", "IncompleteLU", "InteractiveUtils", "IterativeSolvers", "JLD2", "KernelAbstractions", "LinearAlgebra", "Logging", "MPI", "NCDatasets", "OffsetArrays", "OrderedCollections", "PencilArrays", "PencilFFTs", "Pkg", "Printf", "Random", "Rotations", "SeawaterPolynomials", "SparseArrays", "Statistics", "StructArrays"]
-git-tree-sha1 = "ead1d8ec6911ff84ef52c387c827553bbc859c1b"
+path = "/Users/gregorywagner/Projects/Oceananigans.jl"
```

Comment:
I need to change this to test on Gadi; is there a relevant branch in Oceananigans I should be pointing to?

Reply:
`glw-ss/ice-ocean-model`